docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -0,0 +1,66 @@
+# CSI Driver for Amazon EBS configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+The [CSI Driver for Amazon EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+provides a CSI interface used by Container Orchestrators to manage the lifecycle
```

Comment:
Could you please fix the caps here and everywhere else in this document?

Reply:
I'm not sure what you mean here. What should be changed?

Reply:
I think he means this
```suggestion
provides a CSI interface used by container orchestrators to manage the lifecycle
```